### PR TITLE
Pass mapping param on import

### PR DESCRIPTION
**Pull request by YuvalFeldman**

### CODE REVIEW

File: pkg/code/utils.go [hunk 1]
```diff
@@ -235,3 +235,23 @@ func IsCodeParseFailed(codeIntegrationVersion *CodeIntegrationVersion) bool {
 func IsCodeParsing(codeIntegrationVersion *CodeIntegrationVersion) bool {
 	return codeIntegrationVersion.TestStatus == tensorleapapi.TESTSTATUS_DURING_TEST || codeIntegrationVersion.TestStatus == tensorleapapi.TESTSTATUS_BEFORE_TEST
 }
+
+func GetBinderYaml(ctx context.Context, codeIntegrationId string) string {
+	codeIntegrationVersion, err := GetLatestVersion(ctx, codeIntegrationId)
+	if err != nil {
+		return ""
+	}
+	tempDirName := "temp_dss"
```
Comment (asafyehezkel) [inline pkg/code/utils.go:244]: Try to read the tar in memory

File: pkg/code/utils.go [hunk 2]
```diff
@@ -235,3 +235,23 @@ func IsCodeParseFailed(codeIntegrationVersion *CodeIntegrationVersion) bool {
...
+	defer os.RemoveAll(tempDirName)
+
+	if err != nil {
+		return ""
+	}
+
+	content, err := os.ReadFile(tempDirName + "/" + BindingFilePath)
+	if err != nil {
+		return ""
+	}
```
Comment (asafyehezkel) [inline pkg/code/utils.go:255]: add new line here :)

File: pkg/code/utils.go (hunk 1 above)
Comment (YuvalFeldman) [reply pkg/code/utils.go]: Updated

File: pkg/code/utils.go (hunk 2 above)
Comment (YuvalFeldman) [reply pkg/code/utils.go:301]: Updated

File: pkg/code/utils.go [hunk 3]
```diff
@@ -82,6 +88,41 @@ func CloneCodeIntegrationVersion(ctx context.Context, codeIntegrationVersion *te
 	return files, nil
 }
 
+func FetchFileFromTarGz(blobURL, filename string) (string, error) {
```
Comment (asafyehezkel) [inline pkg/code/utils.go]: add type to blobURL

**Review by asafyehezkel — APPROVED** [review]
LGTM

File: pkg/code/utils.go (hunk 3 above)
Comment (YuvalFeldman) [reply pkg/code/utils.go]: Updated

**Pull request merged**